snap/snapcraft.yaml

```diff
@@ -72,6 +72,7 @@ parts:
     plugin: nil
     build-packages:
       - python3-dev
+      - git
```

Comment:
What this is needed for?

Reply:
This was needed to be able to install packages using git+https and can be removed when those lines are removed. I will revert these changes since this brownout period ended.